app/controllers/posts_controller.rb

```diff
@@ -4,6 +4,6 @@ def index
   end
 
   def show
-    @post = Post.find(params[:id])
```

Comment:
- [ ] This controller needs changes because of associations. Posts belong to a user. This request would result in error. The true methods would be like this if we use author and books relation for example:
```
class booksController < ApplicationController
def index 
user = User.find(params[:id])
@books = user.books
end

def show 
user = User.find(params[:id])
@book = Book.find(user, params[:book_id])
end
```

Reply:
Hi @tugsem Thank you for your nice feedback. According the  requirements

-  Add RSpec as a gem to your project.
- Create a Request spec file for all your controllers.
- Make sure that for each action you check:
- If response status was correct.
- If a correct template was rendered.
- If the response body includes correct placeholder text.
It is not necessary to implement this pars .

Reply:
@tugsem Thank you